# cv::Point2f array
# std::vector<int> ids
# every 4 points forms a marker with a id
# 2 points form a face without ids
Header header
geometry_msgs/Point[] points
int32[] ids
uint16 size
